### ISSUE TITLE
[Feature Request] Auto-fill for credit card information

### ISSUE THREAD

**Issue by akanter** [PhilippC/keepass2android#219]
The one thing that keeps me from using auto-fill instead of the Google auto-fill is that I love the credit card entries. Might it be possible to extend the auto-fill service to recognize CC fields and then we could define standard CC entries in the .kdbx? Also if you have any pointers on getting started on this (reasons why it won't work, thoughts on how you might think it could work) please comment here!

**Comment by PhilippC** [comment]
To make this work, there are two requirements: 
1.) Keepass2Android needs to detect the auto-fill fields as Credit Card fields. This will work only if the target app or website uses autofill hints. Can you point me to the app or website where you try to enter the data?
2.) Keepass2Android needs to find an entry with appropriate field names. If you add an entry using the Credit Card Template this should be fine.

**Comment by akanter** [comment]
Certainly! Check out https://www.cryptoheaven.com/secure-web-forms/sample-credit-card-payment-form.htm as an example.

According to a [StackOverflow post](https://stackoverflow.com/questions/15168261/how-does-chrome-detect-credit-card-fields), Chrome actually recognizes it by regexes found [here](https://chromium.googlesource.com/chromium/chromium/+/master/chrome/browser/autofill/autofill_regex_constants.cc.utf8), which might be based off the [WhatWG HTML auto-complete spec](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete).

The credit card template seems perfect and incredibly obvious =P

**Comment by NWH0** [comment]
Any new or planned development concerning this function?

**Comment by bigboj** [comment]
+1

**Comment by 4Misi** [comment]
+1
